### ISSUE TITLE
vectortopdf relies on hard-coded Inkscape CLI arguments that no longer exist

### ISSUE THREAD

**Issue by elloe** [OpenPrinting/cups-filters#315]
This is related to an initial bug report against recent releases of Inkscape on Xubuntu 20.04, where the CUPS braille filter will call `vectortopdf` which has hard-coded Inkscape command line options which have changed.

Inkscape report is [CUPS: Filters: changing inkscape arguments breaks CUPS Braille support](https://gitlab.com/inkscape/inbox/-/issues/3926)

Removed/changed inkscape command-line arguments in recent releases breaks `/usr/lib/cups/filter/vectortopdf` which is used by the **CUPS Braille filter**; specifically here on Xubuntu 20.04.

Inkscape is relied on for the `vectortopdf` shell script which is also sym-linked for several vector image types:
```
$ grep -li inkscape /usr/lib/cups/filter/*
/usr/lib/cups/filter/cgmtopdf
/usr/lib/cups/filter/cmxtopdf
/usr/lib/cups/filter/emftopdf
/usr/lib/cups/filter/svgtopdf
/usr/lib/cups/filter/vectortopdf
/usr/lib/cups/filter/wmftopdf
/usr/lib/cups/filter/xfigtopdf
```
These are mostly called via the braille filter:
```
$ for n in $(grep -li inkscape /usr/lib/cups/filter/* ); do grep -li $(basename $n) /usr/share/cups/mime/*; done
/usr/share/cups/mime/braille.convs
/usr/share/cups/mime/braille.convs
/usr/share/cups/mime/braille.convs
/usr/share/cups/mime/braille.convs
/usr/share/cups/mime/braille.convs
/usr/share/cups/mime/braille.convs
```
#### Steps to reproduce:

Reproduction needs to be a manual operation of reconstructing the command line arguments the `/usr/lib/cups/filter/vectortopdf` script requires from the script:
```
case $0 in
  *svgtopdf*) INPUT_FORMAT=svg ;;
  *xfigtopdf*) INPUT_FORMAT=fig ;;
  *wmftopdf*) INPUT_FORMAT=wmf ;;
  *emftopdf*) INPUT_FORMAT=emf ;;
  *cgmtopdf*) INPUT_FORMAT=cgm ;;
  *cmxtopdf*) INPUT_FORMAT=cmx ;;
esac

FILE_FORMAT=$(mktemp "${TMPDIR:-/tmp}/vectortopdf.XXXXXX.${INPUT_FORMAT}")
FILE_PDF=$(mktemp "${TMPDIR:-/tmp}/vectortopdf.XXXXXX.pdf")

INKSCAPE="inkscape -z --export-area-drawing"
$INKSCAPE -A "$FILE_PDF" "$FILE_FORMAT"
```
Which for an SVG gives us:
```
inkscape -z --export-area-drawing -A /tmp/vectortopdf.XXXXXX.pdf /tmp/vectortopdf.XXXXXX.svg
```
#### What happened?

```
$ inkscape -z --export-area-drawing -A /tmp/vectortopdf.test.pdf ./SLBS-Internet-AGM.svg 
Warning: Option --without-gui= is deprecated
Unknown option -A
```
Inkscape triager reported:

> I believe CUPS needs to check the version of Inkscape installed (calling inkscape -V and checking stdout for 1.0 or later) and call the correct commands (I think reported here? https://github.com/OpenPrinting/cups-filters/issues per https://www.cups.org/doc/reporting-bugs.html)
> 
> imagemagick had to do a similar thing too #496 (closed) https://github.com/ImageMagick/ImageMagick/issues/2062
>

**Comment by tillkamppeter** [comment]
@sthibaul could you have a look here?

**Comment by tillkamppeter** [comment]
Fix is also included in today's 1.28.11 release of cups-filters.
